templates/worker.yaml.tmpl

```diff
@@ -487,9 +493,14 @@ systemd:
       --containerized \
       --enable-server \
       --logtostderr=true \
+      {{if eq .Provider "azure" -}}
+      --cloud-provider=azure \
+      --cloud-config=/etc/kubernetes/config/azure.yaml \
+      {{ else -}}
       --cloud-provider=aws \
+      --pod-infra-container-image={{ .PodInfraImage }} \
```

Comment:
isn't this more like general things rather than aws specific?

Reply:
Yes, it is general. But used only on AWS as we are overriding default for China